Sources/Rendering/Core/Picker/index.js

```diff
@@ -236,8 +236,11 @@ function vtkPicker(publicAPI, model) {
     props.forEach((prop) => {
       const mapper = prop.getMapper();
       pickable = prop.getNestedPickable() && prop.getNestedVisibility();
-      if (prop.getProperty().getOpacity() <= 0.0) {
-        pickable = false;
+
+      if (!mapper.isA('vtkVolumeMapper')) {
```

Comment:
if a volume has an opacity==0, it should probably not be pickable...

Reply:
volumeMapper doesnt have this function

Reply:
I see, then you can simply do:
```
  if (prop.getProperty().getOpacity.?() <= 0.0) {
          pickable = false;
  }
```

Reply:
if (prop.getProperty().getOpacity?.() <= 0.0) {
          pickable = false;
  }

Reply:
Done